pandora-common/src/inputLimits.ts

```diff
@@ -20,6 +20,8 @@ export const LIMIT_MAIL_LENGTH = 256;
 
 /** The maximum amount of characters inside a space */
 export const LIMIT_SPACE_MAX_CHARACTER_NUMBER = 100;
+/** The overshot for admins inside the space */
+export const LIMIT_SPACE_MAX_CHARACTER_ADMIN_OVERSHOT_MULTIPLIER = 1.1;
```

Comment:
Also been thinking about this - multiplier feels a bit strange for the end user. Slightly in-transparent, harder to predict and weird that size of room would affect how many admins can be in extra, especially as the increase points are relatively arbitrary from the user's point of view.

My thoughts are toward making a static, additive overshoot increase instead.

Reply:
Well, I guess if we make the change in the previous comment that it only affects owners for now, we can make it a static 5 for now, but in the future we would want to make this static limit to be user configurable, although we likely would want to still have a max defined, as otherwise you could make the room have 100 slots, plus any number of overshoot for instance for VIPs.